scripts/lisk_bridge.sh

```diff
@@ -57,7 +60,7 @@ extractConfig() {
 # Queries the `/api/loader/status/sync` endpoint
 # and extracts the height for evaluation.
 blockMonitor() {
-  BLOCK_HEIGHT="$(curl -s http://localhost:"$PORT"/api/loader/status/sync | cut -d':' -f 5 | cut -d',' -f 1)"
+	BLOCK_HEIGHT="$(curl -s http://localhost:"$PORT"/api/loader/status/sync | cut -d':' -f 5 | cut -d',' -f 1)"
```

Comment:
Wrong identation?

Reply:
This fixes indentation to be in line with the rest of the file (I should've put that in the commit message)